src/plotman/interactive.py

```diff
@@ -179,21 +174,21 @@ def curses_main(stdscr):
         #
 
         # Directory prefixes, for abbreviation
-        tmp_prefix = os.path.commonpath(dir_cfg['tmp'])
-        dst_prefix = os.path.commonpath(dir_cfg['dst'])
+        tmp_prefix = os.path.commonpath(cfg.directories.dst)
```

Comment:
These sorts of changes are mind numbing.  I get it.  `:]`
```suggestion
        tmp_prefix = os.path.commonpath(cfg.directories.tmp)
```